src/gltf/gltf-parser.ts

```diff
@@ -157,6 +158,36 @@ export class glTFParser {
     return this._materialFactory.create(result)
   }
 
+  /**
+   * Calculates uv transform matrix based on "offset", "rotation" and "scale" 
+   * parameters from extension data (if any), and attaches it to texture
+   * @param KHR_texture_transform extension data from gltf file
+   * @param texture Texture to attach generated matrix to
+   */
+  calculateUVTransform(KHR_texture_transform: any, texture: PIXI.Texture) {	  
```

Comment:
Create an interface for KHR_texture_transform so that we don't have to use "any"